docs/Reference/Pantheon-API-Methods.md

```diff
@@ -533,6 +533,47 @@ None
     }
     ```
 
+### eth_protocolVersion
+
+Returns current Ethereum protocol version. 
+
+**Parameters**
+
+None
+
+**Returns**
+
+`result` : *quantity* - Ethereum protocol version 
+
+!!! example
+    ```bash tab="curl HTTP request"
+    curl -X POST --data '{"jsonrpc":"2.0","method":"eth_protocolVersion","params":[],"id":1}' http://127.0.0.1:8545
+    ```
+    
+    ```bash tab="wscat WS request"
+    {"jsonrpc":"2.0","method":"eth_protocolVersion","params":[],"id":1}
+    ```
+    
+    ```json tab="JSON result"
+    {
+      "jsonrpc" : "2.0",
+      "id" : 1,
+      "result" : 63
+    }
+    ```
+    
+    ```bash tab="curl GraphQL request"
+    curl -X POST -H "Content-Type: application/json" --data '{ "query": "{protocolVersion}"}' http://localhost:8547/graphql
+    ```
+        
+    ```json tab="GraphQL result"
+    {
+      "data" : {
+        "protocolVersion" : 63
```

Comment:
This is correct, because GraphQL spec is different.

Reply:
No change needed